blaze-server/src/main/scala/org/http4s/server/blaze/Http2NodeStage.scala

```diff
@@ -199,7 +200,7 @@ class Http2NodeStage(streamId: Int,
       val hs = HHeaders(headers.result())
       val req = Request(method, path, HttpVersion.`HTTP/2.0`, hs, body, attributes)
 
-      Task.fork(service(req)).runAsync {
+      Task.fork(service(req))(executor).runAsync {
```

Comment:
0.12.2 possibility: we can turn an `ExecutionContext` into an `ExecutorService` that just doesn't close.